driver-core/src/test/functional/com/mongodb/operation/ChangeStreamOperationSpecification.groovy

```diff
@@ -331,6 +331,33 @@ class ChangeStreamOperationSpecification extends OperationFunctionalSpecificatio
         waitForLastRelease(getCluster())
     }
 
+    @IgnoreIf({ !serverVersionAtLeast([4, 0, 1]) })
+    def 'should decode dropDatabase to ChangeStreamDocument '() {
```

Comment:
This test doesn't pass on this commit.  It relies on a subsequent commit that fixes the codec.  It's generally good practice to have even intermediate commits pass the tests, so before you push this to master you should either squash all commits or re-order them so the fixes to to the codec are first, which will ensure that this test passes.

Reply:
Okay, I'll keep that in mind. I'll make sure to squash the commits before I push to master!